packages/react-core/src/components/EmptyState/EmptyStateIcon.tsx

```diff
@@ -9,20 +9,11 @@ export enum IconSize {
   xl = 'xl'
 }
 
-export interface IconProps extends Omit<React.HTMLProps<SVGElement>, 'size'> {
-  /** deprecated */
-  color?: string;
-  /** deprecated */
-  size?: 'sm' | 'md' | 'lg' | 'xl';
-  /** deprecated */
-  title?: string;
-}
-
-export interface EmptyStateIconProps extends IconProps {
+export interface EmptyStateIconProps {
   /** Additional classes added to the EmptyState */
   className?: string;
   /** Icon component to be rendered inside the EmptyState on icon variant */
-  icon?: string | React.FunctionComponent<IconProps>;
+  icon?: string | React.FunctionComponent<any>;
```

Comment:
I'm still warming up my brain from my leave. Would we lose some of the ability to pass props and set aria-hidden to true on line 32 if we let them pass a node? It would change the return statement a bit to allow a ReactNode rather than a React.FunctionComponent, right?

Reply:
Yep that's correct @nicolethoen ! We'd have to do something like `{React.isValidElement(icon) ? React.cloneElement(icon, { className: "", ...props, 'aria-hidden': true }) : icon}`.

Since we're expecting an icon (and not a string) I think `React.ComponentType<any>` and leaving it as-is after #4069 is also fine. Supporting a string just requires that extra `React.isValidElement` check (I'm not sure how `React.cloneElement` works on strings).